### DISCUSSION TITLE
Doks 1.4 is now available!

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#1195] [Announcements]
Doks 1.4 is now available! This release includes a new shortcode for link cards, improved sidebar navigation, and more.

Highlights include:

- **Link cards shortcode**: You can now use the `{{< link-card >}}` shortcode to link prominently to different pages.
- **Improved sidebar navigation**:  By default, Doks will automatically generate a sidebar based on the filesystem structure of your documentation.

Read more in the [blog post announcement](https://getdoks.org/blog/doks-1-4/)

**Comment by cnichte** [comment]
Love the Link-Card, and the fixes. I built a simple  [Linktree](https://carsten-nichte.de/linktree/) with the link-cards, [like this one](https://linktr.ee).  In addition, my Sidebar works finally like expected! great update!

**Reply by h-enk** [reply]
Thanks – nice to hear! Love your link tree – that's a great example of what you can do with link cards
